### PR TITLE
docs(styling): some info about the way to customize design

### PR DESCRIPTION
**Pull request by warlof**

### CODE REVIEW

File: docs/styling.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+![SeAT](https://i.imgur.com/aPPOxSK.png)
+
+# Styling
+
+By default, SeAT is shipped using Bootstrap 3 and the template Admin LTE.
```
Comment (leonjza) [inline docs/styling.md]: > SeAT uses Bootstrap 3 and the [Admin LTE](https://adminlte.io/) template.

File: docs/styling.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+You may want to customise a bit your own SeAT design to either your corporation or alliance colors.
```
Comment (leonjza) [inline docs/styling.md]: > You may want to customise SeAT design to match either your corporation or alliance colours.

File: docs/styling.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+To doing so, you can use two available css hooks :
```
Comment (leonjza) [inline docs/styling.md]: > To do so, you can use two available css hooks :

File: docs/styling.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+ - `custom-layout-mini.css` which is used by the sign-in page
+ - `custom-layout.css` which is used for all the application
```
Comment (leonjza) [inline docs/styling.md]: > used by the sign-in page

> used by all the entire application, globally

File: docs/styling.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+Both of them must be located into your `public` directory.
+
+!!! exemple
```
Comment (leonjza) [inline docs/styling.md]: > Example

File: docs/styling.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    Using the default base directory, you'll get the following path :
+    - `/var/www/seat/public/custom-layout-mini.css`
+    - `/var/www/seat/public/custom-layout.css`
+
+Those file are loaded automatically by the core if detected - you have nothing else to do to enable them.
```
Comment (leonjza) [inline docs/styling.md]: These files are loaded automatically if they are detected - you have nothing else to do to enable them.

File: docs/styling.md (hunk 5 above)
Comment (warlof) [reply docs/styling.md]: this one is an admonition render :)
https://squidfunk.github.io/mkdocs-material/extensions/admonition/#example

**Comment by warlof** [comment]
Tong has been fixed.

I also add references for both Admin LTE and Bootstrap :)

File: docs/styling.md (hunk 5 above)
Comment (leonjza) [reply docs/styling.md]: Just spell it as example and not exemple.

Comment (warlof) [reply docs/styling.md]: oh, my bad 🙄

**Pull request merged**